### ISSUE TITLE
Explore using Loomio for asynchronous decision making

### ISSUE THREAD

**Issue by laveshaparker** [ProgressiveHackNight/admin#15]
https://www.loomio.org/

[High-level explanation of Loomio on YouTube](https://www.youtube.com/watch?v=B1gLEAKPkOc)

**Comment by laveshaparker** [comment]
*tl;dr*: I love this

### Findings
- We can use this for $00.free!!!!
- We won't have slack integration with the free version, but ¯\_(ツ)_/¯
- You can set vote reminders
- You can set a decision to notify participants
- Their proposal feature loosely matches our current +1, 0, -1 voting setup:
  - >Participants can agree, abstain, disagree or block - and give a statement to explain their position.
- A *super cool* feature is to have different types of decisions available:
<img width="485" alt="screen shot 2017-05-30 at 9 46 40 pm" src="https://cloud.githubusercontent.com/assets/5737949/26612231/8c0d79f6-4581-11e7-9e47-20db5b5888f0.png">


#### Notable excerpts from Terms and Conditions
> As a user, you may deactivate your account at any time. Identifying information such as your username and profile photo will no longer be associated with content you have posted. However, the content itself will remain [...].

>Users and group coordinators may request data related to their personal account and group(s) respectively. Loomio will take reasonable efforts to provide these data. Loomio has the right to refuse repetitive or technically difficult requests.

#### Notable excerpts from Privacy Policy
>Loomio collects non-personally-identifying information [...] such as the browser type, language preference, referring site, and the date and time of each visitor request

>Loomio also collects potentially personally-identifying information like Internet Protocol (IP) addresses. 

>We also collect the name and email address users log in with. [...] you are welcome to use a pseudonym and anonymous email address to prevent this.

>Loomio will not sell or rent any information you share with us. Access to your content and personally identifiable information is tightly controlled internally. We will never view your private content without your consent.

>To provide some of Loomio’s functionality and a high level of service, Loomio shares user data with third parties. These third parties adhere to industry best practices meaning that they in turn will not sell any data shared with them.

**Comment by rapicastillo** [comment]
Voting +1 to this! <3

**Comment by julianrad** [comment]
+1

**Comment by tktino** [comment]
+1

**Comment by laveshaparker** [comment]
Cool! Looks like we'll move forward with this.
